### PR TITLE
Move Code Reviewers from Wiki into the book

### PR DESCRIPTION
**Pull request by ntindle**
Moving code reviewers list from the wiki into the book both makes the list of reviewers more public while also allowing people to add themselves. This will help with reducing the workload of maintainers while also allowing automation later to access the data and automatically assign reviews to those listed by each language when a label is assigned.

Note, due to the oddity with honkit markdown, all usernames must be separated by a character that isn't used in link notation. I've chosen "," (comma) as it is easy to parse and a common list seperator.

### CODE REVIEW

File: contents/code_reviews/code_reviewers.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+[//]: # (To add yourself to the table, please add a link at the bottom of the article and reference yourself in the following format next to each language [@username]. If there are multiple usernames for a language, make sure to seperate them with commas)
+
+Language | Reviewers
+-|-
+asm |
+c |
+c# |
+clojure |
+coconut | [@amaras]
+c++ |
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:14]:
```suggestion
c++ | [@ShadowMitia]
```

File: contents/code_reviews/code_reviewers.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+julia | [@leios]
+kotlin |
+lisp |
+lua |
+matlab |
+nim |
+php |
+python | [@ntindle]
+ruby |
+rust |
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:31]:
```suggestion
rust | [@ShadowMitia]
```

File: contents/code_reviews/code_reviewers.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+clojure |
+coconut | [@amaras]
+c++ |
+emojicode |
+fortran | [@leios]
+gnuplot | [@leios]
+go |
+haskell |
+java |
+javascript | [@ntindle]
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:21]:
```suggestion
javascript | [@ntindle] [@ShadowMitia]
```

File: contents/code_reviews/code_reviewers.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+julia | [@leios]
+kotlin |
+lisp |
+lua |
+matlab |
+nim |
+php |
+python | [@ntindle]
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:29]:
```suggestion
python | [@ntindle] [@ShadowMitia]
```

**Review by ShadowMitia — COMMENTED** [review]
Adding myself to a few languages I'm confident I can help

File: contents/code_reviews/code_reviewers.md (hunk 3 above)
Comment (ntindle) [reply contents/code_reviews/code_reviewers.md:21]: can you separate with a comma

File: contents/code_reviews/code_reviewers.md (hunk 4 above)
Comment (ntindle) [reply contents/code_reviews/code_reviewers.md:29]: can you separate with a comma

Comment (ShadowMitia) [reply contents/code_reviews/code_reviewers.md:29]: Like this?

File: contents/code_reviews/code_reviewers.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+go |
+haskell |
+java |
+javascript | [@ntindle],[@ShadowMitia]
+julia | [@leios]
+kotlin |
+lisp |
+lua |
+matlab |
+nim |
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:27]:
```suggestion
nim |
ocaml | [@ShadowMitia]
```

File: contents/code_reviews/code_reviewers.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
+# Code Reviewers
+
+If you are comfortable reviewing a language, please add yourself to the table below:
+
+[//]: # (To add yourself to the table, please add a link at the bottom of the article and reference yourself in the following format next to each language [@username]. If there are multiple usernames for a language, make sure to seperate them with commas)
+
+Language | Reviewers
+-|-
+asm |
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:9]:
```suggestion
asm |
bash |
```

File: contents/code_reviews/code_reviewers.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+c |
+c# |
+clojure |
+coconut | [@amaras]
+c++ | [@ShadowMitia]
+emojicode |
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:15]:
```suggestion
c |
c# |
clojure |
coconut | [@amaras]
c++ | [@ShadowMitia]
crystal |
D |
dart |
elm |
emojicode |
```

File: contents/code_reviews/code_reviewers.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+fortran | [@leios]
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:16]:
```suggestion
factor |
fortran | [@leios]
```

File: contents/code_reviews/code_reviewers.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+gnuplot | [@leios]
+go |
+haskell |
+java |
+javascript | [@ntindle],[@ShadowMitia]
+julia | [@leios]
+kotlin |
+lisp |
+lua |
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:25]:
```suggestion
kotlin |
labview |
lolcode |
lisp |
lua |
```

File: contents/code_reviews/code_reviewers.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+matlab |
+nim |
+php |
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:28]:
```suggestion
php |
piet |
powershell |
```

File: contents/code_reviews/code_reviewers.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+python | [@ntindle],[@ShadowMitia]
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:29]:
```suggestion
python | [@ntindle],[@ShadowMitia]
r |
racket |
```

File: contents/code_reviews/code_reviewers.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+ruby |
+rust | [@ShadowMitia]
+scala |
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:32]:
```suggestion
scala |
scheme |
```

File: contents/code_reviews/code_reviewers.md [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+scratch | [@leios]
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:33]:
```suggestion
scratch | [@leios]
smalltask |
```

File: contents/code_reviews/code_reviewers.md [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+swift |
+typescript | [@ntindle]
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:35]:
```suggestion
typescript | [@ntindle]
v |
```

**Review by ShadowMitia — CHANGES_REQUESTED** [review]

File: contents/code_reviews/code_reviewers.md [hunk 15]
```diff
@@ -21,18 +27,28 @@ java |
...
+lolcode |
 lisp |
 lua |
 matlab |
 nim |
+ocaml | [@ShadowMitia]
 php |
+piet |
+powershell |
 python | [@ntindle],[@ShadowMitia]
```
Comment (Amaras) [inline contents/code_reviews/code_reviewers.md:40]:
```suggestion
python | [@ntindle],[@ShadowMitia],[@amaras]
```

**Review by Amaras — COMMENTED** [review]
Adding myself now

File: contents/code_reviews/code_reviewers.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+nim |
+ocaml | [@ShadowMitia]
+php |
+piet |
+powershell |
+python | [@ntindle],[@ShadowMitia],[@amaras]
+r |
+racket |
+ruby |
+rust | [@ShadowMitia]
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:44]:
```suggestion
rust | [@ShadowMitia],[@PeanutbutterWarrior]
```

File: contents/code_reviews/code_reviewers.md [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+lolcode |
+lisp |
+lua |
+matlab |
+nim |
+ocaml | [@ShadowMitia]
+php |
+piet |
+powershell |
+python | [@ntindle],[@ShadowMitia],[@amaras]
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:40]:
```suggestion
python | [@ntindle],[@ShadowMitia],[@amaras],[@PeanutbutterWarrior]
```

File: contents/code_reviews/code_reviewers.md [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+
+If you are comfortable reviewing a toolchain change, please add yourself to the list below:
+
+Feature | Reviewers
+-|-
+Dev Container | [@ntindle], [@ShadowMitia]
+Docker | [@ntindle], [@ShadowMitia]
+Github Actions | [@ntindle]
+Honkit |
+Scons | [@amaras]
```
Comment (ShadowMitia) [inline contents/code_reviews/code_reviewers.md:63]:
```suggestion
Scons | [@amaras],[@PeanutbutterWarrior]
```

**Review by ShadowMitia — APPROVED** [review]

**Pull request merged**